_inc/lib/class-jetpack-podcast-helper.php

```diff
@@ -166,8 +179,7 @@ private static function get_audio_enclosure( SimplePie_Item $episode ) {
 			}
 		}
 
-		// Default to empty SimplePie_Enclosure object.
-		return $episode->get_enclosure();
+		return new WP_Error( 'invalid_audio', __( 'Podcast audio is an invalid type.', 'jetpack' ) );
```

Comment:
Is there a reason we return `WP_Error` here rather than `null`, like `SimplePie_Item::get_enclosure()`? The error doesn't seem to see the light of day either

Reply:
Yes we should have returned null particularly as the error isn't exposed. 

Did you create a followup Issue for this yet?